plugins/modules/rpm_repository.py

```diff
@@ -103,6 +125,16 @@ def main():
             # In case of an empty string we nullify the description
             desired_attributes["description"] = module.params["description"] or None
 
+        if module.params["remote"] is not None:
+            remote = PulpRpmRemote(module, {"name": module.params["remote"]})
+            remote.find(failsafe=False)
+            desired_attributes["remote"] = remote.href
```

Comment:
Too bad this module wasn't yet translated to using glue...
I hope that won't become more complicated now.